data/saas/config/greenhouse_config.yml

```diff
@@ -0,0 +1,66 @@
+saas_config:
+  fides_key: greenhouse_connector_example
+  name: Greenhouse SaaS Config
+  description: A sample schema representing the Greenhouse connector for Fidesops
+  version: 0.0.1
+
+  connector_params:
+    - name: user_id
+    - name: username
+    - name: email
```

Comment:
Remove email identity, same reasoning as the Gorgias connector https://github.com/ethyca/fidesops/pull/304/files#r830290394

Reply:
Done